target/linux/ramips/image/mt7621.mk

```diff
@@ -335,6 +335,17 @@ define Device/iodata_wnpr2600g
 endef
 TARGET_DEVICES += iodata_wnpr2600g
 
+define Device/iptime_a8004t
+  MTK_SOC := mt7621
+  IMAGE_SIZE := 16064k
```

Comment:
16064k is 0xfb0000, which does not match with the firmware partition size in dts.

Reply:
Isn't this 0xfc0000? because 0x40000 + 0xfc0000 = 0x1000000 = 16MiB

Reply:
0xfc0000 is 16128k, so you should change IMAGE_SIZE accordingly.

Reply:
OK, Fixed.

Reply:
As you say:
16 MiB = 16384 kiB = 0x1000000 != 0xfc0000 = 16128 kiB